Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +42,36 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            var stop = 0;
+
+            if (array == null)
+            {
+                return;
+            }
+
+            stop = array.Length / 2;
+
+            // Here I take a element and put it into temp. 
+            // So I can put there the element of the other half of the arrau
+            // and put that on that place. After that I can take the contents of
+            // temp and put it on the new place 
+
+            // Example lets say we have this {1,2,3,4,5} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887669)